src/main/services/i18n/locales/zh/special.json

```diff
@@ -0,0 +1,24 @@
+{
+  "description": {
+    "storage": "要使用iCloud Drive、Google Drive和Dropbox等同步服务，只需将存储路径设置为相应的同步文件夹即可。",
+    "migrate": {
+      "1": "从massCode v1.0迁移：选择包含数据库文件的文件夹。",
+      "2": "从SnippetsLab迁移：选择JSON文件。",
+      "3": [
+        "从SnippetsLab迁移将会遇到一些局限：",
+        "所有的文件夹将是第一层，因为JSON文件（低于v2.1）无法表示嵌套的文件夹。",
+        "某些片段可能会因为其语言暂时不被massCode而被识别为纯文本。"
+      ]
+    },
+    "htmlCssPreview": "添加带有HTML和CSS语言的片段来查看预览结果。"
+  },
+  "success": {
+    "migrate": "数据库迁徙成功。"
+  },
+  "error": {
+    "folderNotContainDb": "文件夹内不存在 \"db.json\"."
+  },
+  "unsponsored": "未赞助的",
```

Comment:
这个地方我本来想翻译为“下次一定”，我以为是对话框的按钮，但是看到显示的位置有些微妙 😂

Reply:
我也觉得有点微妙就暂时直译了。
我觉得你的那个创建副本翻译的太好了。我是按照macOS的访达的逻辑翻译成复制的，而且同一个右键菜单里面没有copy就不会混淆——但是访达这个拷贝和复制我第一次遇到就懵逼哈哈，你的创建副本好太多。

Reply:
> 我也觉得有点微妙就暂时直译了。 我觉得你的那个创建副本翻译的太好了。我是按照macOS的访达的逻辑翻译成复制的，而且同一个右键菜单里面没有copy就不会混淆——但是访达这个拷贝和复制我第一次遇到就懵逼哈哈，你的创建副本好太多。

想了很久才想起来这个最合适的，不清楚他能不能 get 到“下次一定” 😁

Reply:
“下次一定” Direct translation is ‘’next time definitely‘’.
Chinese internet buzzwords, A video site from bilibili. It is said that a certain UP owner prayed to the audience for three tasks at the end of the video (like, coin, and favorite), and one audience posted "下次一定". Because there are many UP masters praying for, the audience finds this polite remark very useful. Therefore, everyone often hits "下次一定" on the barrage to deal with it perfectly, save the face of both parties, and then continue to hit "下次一定" next time.
We commonly call this type of friends "white whore monsters". Of course, there will be many people who will meet the request of the UP master.

Reply:
@sangyuxiaowu I can't translate or make sense of it. Are you amused by the fact that it says "Unsponsored" in the app? )

Reply:
> @sangyuxiaowu I can't translate or make sense of it. Are you amused by the fact that it says "Unsponsored" in the app? )

Of course not, it's not funny, just for translation. Open source has helped everyone, and everyone should provide support and help for you. Later, I will continue to help you promote the project in the Chinese community, so that more people can understand and help the project do better.

Reply:
> @sangyuxiaowu I can't translate or make sense of it. Are you amused by the fact that it says "Unsponsored" in the app? )

I'm sorry to make you misunderstand. It is indeed a social phenomenon that people are unwilling to pay. My words are also translated. With cultural differences, the misunderstanding may be deeper.

Reply:
Thanks for the understanding and support

Reply:
@sangyuxiaowu 说起来这个显示在哪里 我一直没找到